src/data/mv_polynomial/basic.lean

```diff
@@ -419,6 +419,10 @@ by rw [X_pow_eq_monomial, support_monomial, if_neg (one_ne_zero' R)]
 
 @[simp] lemma support_zero : (0 : mv_polynomial σ R).support = ∅ := rfl
 
+lemma support_smul [distrib_mul_action R S₁] {a : R} {f : mv_polynomial σ S₁} :
```

Comment:
I think `a` and `f` should be explicit here, unless you have a reason to keep them implicit.

Reply:
I am matching `finsupp.support_smul`, which has them implicit. And I am happy to keep it like that because I noticed that we tend to never use the explicit arguments of `≤`, `<`, `⊆`, `⊂` lemmas.